routes/lang/nl-application.json

```diff
@@ -0,0 +1,328 @@
+  "app.password": "wachtwoord",
+  "app.login": "Log in",
+  "app.logout": "Log uit",
+  "app.signin": "Meld aan",
+  "app.help": "Help",
+  "app.register": "Registreer",
+  "app.pleaseloginor": "Log in aub, of",
+  "app.invalidusernameoremail": "Incorrect e-mail adres",
+  "app.invalidusernameorpassword": "Incorrect e-mail adres of wachtwoord",
+  "app.accountlockedout": "Uw account is geblokkeerd, neeem aub contact op met de administrator voor deblokkade",
+  "app.areyousureyouwantto": "Weet u zeker dat u",
+  "app.dashboard": "Dashboard",
+  "app.loggedinas": "Ingelogd als",
+  "app.edit": "Bewerken",
+  "app.view": "Bekijk",
+  "app.restore": "Herstellen",
+  "app.editSettings": "Bewerk insellingen",
+  "app.editCourse": "Bewerk cursus",
+  "app.editproperties": "Bewerk eigenschappen",
+  "app.duplicate": "Dupliceer",
+  "app.delete": "Verwijder",
+  "app.copy": "Kopie",
+  "app.copyidtoclipboard": "Kopieer ID naar klembord",
+  "app.paste": "Plakken",
+  "app.cut": "Knippen",
+  "app.save": "Opslaan",
+  "app.saving": "Opslaan...",
+  "app.saveandedit": "Opslaan en bewerken",
+  "app.continue": "Doorgaan",
+  "app.cancel": "Annuleren",
+  "app.confirmduplicate": "Weet u zeker dat u deze cursus wilt dupliceren om lokaal te bewerken?",
+  "app.myprojects": "Mijn cursussen",
+  "app.addnewproject": "Voeg nieuwe cursus toe",
+  "app.sharedprojects": "Gedeelde curssusen",
+  "app.noprojectstodisplay": "Geen cursus weer te geven",
+  "app.pleaseentervalue": "Voer hier aub een waarde in",
+  "app.pleaseaddfile": "Selecteer een bestand voor upload",
+  "app.projecttitle": "Project titel",
+  "app.projectdescription": "Project beschrijving",
+  "app.addarticle": "Voeg artikel toe",
+  "app.addblock": "Voeg blok toe",
+  "app.addcomponent": "Voeg component toe",
+  "app.deletearticle": "Verwijder artikel",
+  "app.deleteblock": "Verwijder blok",
+  "app.deletecomponent": "Verwijder component",
+  "app.pleaseselectcomponent": "Selecteer aub een component",
+  "app.addtoleft": "Links toevoegen",
+  "app.addfullwidth": "Voeg toe",
+  "app.addtoright": "Rechts toevoegen",
+  "app.typetofiltercomponents": "Typ om op componenten te filteren",
+  "app.addpage": "Voeg pagina toe",
+  "app.addsection": "Voeg sectie toe",
+  "app.editthistext": "{ Bewerk deze tekst}",
+  "app.overview": "Overzicht",
+  "app.resetpassword": "Reset wachtwoord",
+  "app.resetpasswordsuccess": "Uw wachtwoord is veranderd",
+  "app.resetpassworderror": "Uw wachtwoord kon niet veranderd worden",
+  "app.submit": "Zenden",
+  "app.newpassword": "Nieuw wachtwoord",
+  "app.forgotpasswordblurb": "Voer het e-mail adres in wat bij uw Learning Pool Adapt account hoort, klik dan op Continue. Wij sturen een email met een link waar u makkelijk een nieww wachtwoord maakt.",
+  "app.forgotpasswordfooter": "Indien u niet langer het e-mail adres wat bij uw  Adapt account hoort gebruikt, neem dan contact op met Support voor hulp bij het herstellen van uw account.",
```

Comment:
I was not sure if it is still valid, so I left it in. Might it be possible to change it with something like 'your organization' ?